swiftsimio/visualisation/projection_backends/reference.py

```diff
@@ -38,6 +38,7 @@ def scatter(x: float64, y: float64, m: float32, h: float32, res: int) -> ndarray
     for a 25-50% performance improvement. In our testing, using numpy
     floats and integers is also an improvement over using the numba ones.
     """
+    # ALEXEI: add param, return, examples docs
```

Comment:
Please just ask users to refer to the documentation in `fast` for parameters or we can do:
```
from swiftsimio.visualisation.projection_backends.fast import scatter as scatter_fast
from swiftsimio.visualisation.projection_backends.fast import scatter_parallel as scatter_fast_parallel
```

and after the definition of `scatter` and `scatter_parallel`:

```
scatter.__doc__ = scatter_fast.__doc__
scatter_parallel.__doc__ = scatter_fast_parallel.__doc__
```

which do you think is best?